### ISSUE TITLE
Static (JS-free) version of the widget

### ISSUE THREAD

**Issue by captainbrosset** [web-platform-dx/baseline-status#35]
In https://github.com/web-platform-dx/baseline-status/issues/16#issuecomment-2456583168, I mentioned that we should investigate ways to offer the baseline widget in a static (JS-less) way too.

See also [Baseline status of a web platform feature on a Hugo website](https://pawelgrzybek.com/baseline-status-of-a-web-platform-feature-on-a-hugo-website/), which is somewhat related.

**Comment by niutech** [comment]
Here is one implementation using Declarative Shadow DOM: [Astro Embed Baseline Status](https://github.com/delucis/astro-embed/tree/main/packages/astro-embed-baseline-status)
But making a pure CSS/HTML Baseline Status widget without DSD should be even easier, provided that the API responds with HTML in addition to JSON.

**Comment by devnook** [comment]
If I understand your intent correctly, you'd like to move the fetching of the compatibility data to the backend / the build step of a static website. The problem with this approach is that if the page does not get rebuilt on regular basis, the Baseline status displayed by the widget will not get updated and will show stale information. Adding an "status as of <some-date-here>" annotation could mitigate that a little, however the UX will not be great with the widget displaying misleading info.

**Comment by niutech** [comment]
@devnook Yes, processing the data on backend. For example, fetching the baseline status as an SVG image with URL like `https://api.webstatus.dev/v1/features/*?format=svg` or as an `<iframe>` with `src` like `https://api.webstatus.dev/v1/features/*?format=html` featuring just basic HTML/CSS for older browsers, no JS.

**Comment by devnook** [comment]
It would need to be implemented on the webstatus.dev side then. Given that webstatus already exposes an API, from technical side this should be well possible. Can you raise an issue for this in the https://github.com/GoogleChrome/webstatus.dev repo?  You can cross-link to this discussion to keep the context.

**Comment by captainbrosset** [comment]
Closing this issue based on the above conversation. Feel free to comment if you think this should be re-opened.
